### DISCUSSION TITLE
Why are there so many chunks and what do they mean?

### DISCUSSION THREAD

**Discussion by afzalsayed96** [vercel/next.js#11823] [Help]
I have set up a basic example with next.js and tailwind-css at [afzalsayed96/next-with-tailwindcss
](https://github.com/afzalsayed96/next-with-tailwindcss) and deplyed [here](https://next-js.afzal4.now.sh/)

The page loads the following js:

Name|Size
-- | -- 
index.js | 2.2 kB 
_app.js | 567 B 
webpack-b65cab0b00afd201cbda.js | 953 B 
framework.98c1b221acb34aa9927b.js | 42.1 kB 
0a471c80.37dff8ebba14344d5875.js | 224 B 
commons.24943936a43f226f9a8e.js | 11.3 kB 
main-fe9ec89c5cfe8f7ed988.js | 6.4 kB 
_buildManifest.js | 233 B 
_ssgManifest.js | 229 B 

The next.js app has only one route `/index` and one catch-all route `/[...rest]` to work with a CMS. Would it be possible to set up a custom webpack splitChunks strategy to optimize for this type of configuration?

**Comment by puntope** [comment]
You can extend your own wepack config here.

https://nextjs.org/docs/api-reference/next.config.js/custom-webpack-config

**Comment by afzalsayed96** [comment]
@puntope I understand. Though first I would need to figure what the individual chunks are for.

For instance do `_app.js` and `index.js` need to be separate? Or why is the empty 61 Byte `0a471c80.37dff8ebba14344d5875.js` chunk generated?

Is there any resource that explains this?

**Reply by ruXlab** [reply]
Hey, did you figure it out? I found it's that _buildManifest and _ssgManifest are not bundled in with any js files, or they could even be placed inside SSG page
